python/dllib/src/bigdl/dllib/utils/zoo_engine.py

```diff
@@ -112,8 +112,10 @@ def get_analytics_zoo_classpath():
     """
     if os.getenv("BIGDL_CLASSPATH"):
         for path in os.getenv("BIGDL_CLASSPATH").split(":"):
-            if not os.path.exists(path):
-                raise ValueError("Path {} specified BIGDL_CLASSPATH does not exist.".format(path))
+            for pa in path.split("*"):
```

Comment:
why need to split by *?
change pa to a more meaningful name? Also add some comment for others to understand what this code segment is doing?

Reply:
`export BIGDL_CLASSPATH=/opt/bigdl-0.14.0-SNAPSHOT/jars/*`.  However, `os.path.exists("/opt/bigdl-0.14.0-SNAPSHOT/jars/*") `return false.

Reply:
okay. But init on k8s is for pip install, in this case, uses don't need to set BIGDL_CLASSPATH right? In our code, I think there won't be `*`?

Reply:
Yes. If using pip install, the path is like `/usr/local/envs/pytf1/lib/python3.7/site-packages/bigdl/share/dllib/lib/bigdl-dllib-spark_2.4.6-0.14.0-SNAPSHOT-jar-with-dependencies.jar:/usr/local/envs/pytf1/lib/python3.7/site-packages/bigdl/share/orca/lib/bigdl-orca-spark_2.4.6-0.14.0-SNAPSHOT-jar-with-dependencies.jar:`
However, k8s image is default built on spark312, which may need download whl from Sourceforge when building image. And k8s cluster need spark-submit. In order to manage the image easily for both mode, we prefer to use the BIGDL_CLASSPATH which keeps the same way with original analytics-zoo k8s image?